keystore/src/connection/platform/wasm/mod.rs

```diff
@@ -99,113 +101,29 @@ impl DatabaseConnection for WasmConnection {
         // - build: breaks after r9
         let version = version_number(version_major, version_minor, version_patch, version_pre);
```

Comment:
I realize this is out of scope for this PR, but nevertheless I'd like to mention here that we should completely decouple the db schema version from the core-crypto version. The db version should be a simple counter,  with no semantic connection to core-crypto version, like the generic platform has it.